### DISCUSSION TITLE
Wrong "csi_rs_period" in configuration reference

### DISCUSSION THREAD

**Discussion by arman-maghsoudnia** [srsran/srsRAN_Project#262] [Ideas & Suggestions]
Hi, 

In the Configuration References page `(https://docs.srsran.com/projects/project/en/latest/user_manuals/source/config_ref.html)`, in the `cell_cfg:csi:csi_rs_period` field, it is mentioned that the default value for `csi_rs_period` is `80ms`. But I checked the code in `apps/gnb/gnb_appconfig.h:320`, and the default value is `20ms`. Could you fix it?

Best,
Arman

**Comment by brendan-mcauliffe** [comment]
Hi @arman-maghsoudnia I've made this change in the latest update to the docs. Thanks for catching it!
